server/handlers/cdn-caching.js

```diff
@@ -2,18 +2,26 @@ const _ = require("lodash");
 
 exports.addCacheHeadersToResult = function addCacheHeadersToResult(
   res,
-  cacheKeys
+  cacheKeys,
+  cdnProvider = "cloudflare"
 ) {
   if (cacheKeys) {
     if (cacheKeys === "DO_NOT_CACHE") {
       res.setHeader("Cache-Control", "private,no-cache,no-store,max-age=0");
+      cdnProvider === "akamai" &&
+        res.setHeader("Edge-Control", "private,no-cache,no-store,max-age=0");
       res.setHeader("Vary", "Accept-Encoding");
       res.setHeader("Surrogate-Control", "private,no-cache,no-store,max-age=0");
     } else {
       res.setHeader(
         "Cache-Control",
         "public,max-age=15,s-maxage=900,stale-while-revalidate=1000,stale-if-error=14400"
       );
+      cdnProvider === "akamai" &&
```

Comment:
comparison `cdnProvider === "akamai"` is being subjected to a total of 4 times. Can we get this out to variable and use it across? (DRY + Perf)